include/glow/Partitioner/Partitioner.h

```diff
@@ -161,6 +164,11 @@ class Partitioner final : public PartitionerBase {
   /// for function optimization. \returns the partition result or an error.
   llvm::Expected<DAGListTy> heterogeneousPartition(CompilationContext &cctx);
 
+  /// Performs a load balancing optimization pass to optimize for load ///
```

Comment:
fix comment

Reply:
can you add more details about "load balance", what exactly does that mean in the context of partitioner.